app/main.py

```diff
@@ -1,3 +1,17 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+import functools
+
+
+def cache(func: Callable) -> Any:
```

Comment:
`cache` return type annotation should be `Callable`, not `Any`, because `inner` is always callable.